src/Middleware/Spa/SpaServices.Extensions/src/AngularCli/AngularCliMiddleware.cs

```diff
@@ -27,15 +27,8 @@ internal static class AngularCliMiddleware
         var pkgManagerCommand = spaBuilder.Options.PackageManagerCommand;
         var sourcePath = spaBuilder.Options.SourcePath;
         var devServerPort = spaBuilder.Options.DevServerPort;
-        if (string.IsNullOrEmpty(sourcePath))
-        {
-            throw new ArgumentException("Property 'SourcePath' cannot be null or empty", nameof(spaBuilder));
```

Comment:
The new exception message is now:

> System.ArgumentNullException: Value cannot be null. (Parameter 'sourcePath')

As with `CascadingValue<TValue>.SetParametersAsync`, it's worth calling out that 'SourcePath' is a property.

The new message references `sourcePath` which comes from a local variable and does not match the casing of any part of the public API. We should never be passing local variable names into these `ArgumentException` helpers.

Property names are okay only if that matches what we were doing before, since at least that's not making things worse. But here, the old message was better since it did message that it was a "Property" that was null.

If I missed other places where I missed we're making the exception message less detailed, please revert those too.

Reply:
Reverted!